source/cpp/korteweg-de-vries/README.md

```diff
@@ -9,7 +9,7 @@ To view data generated:
 ```
 $ conda create -n kdv
 $ conda activate kdv
-$ conda install -c williamfgc adios2-nompi -c conda-forge
+$ conda install -c williamfgc adios2 -c conda-forge
```

Comment:
My bad, I should have been more clear.  Try testing with the conda-forge one `conda install adios2 -c conda-forge`, the `williamfgc` channel is for some Jupyter-notebooks and quick access to build types.

Reply:
Strange error with that one:

```
conda install adios2 -c conda-forge
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.

(kdv) ➜  korteweg-de-vries git:(kdv2) ✗ python3 -q
>>> import adios2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'adios2'
```